src/processHooks.ts

```
@@ -58,11 +63,16 @@ function installGlobalShutdownHooksOnce() {
         activeDisposers.delete(d);
       }
     } finally {
-      process.exit(0);
+      try {
+        platform.exit(0);
+      } catch (e) {
+        if (!(e instanceof PlatformUnsupportedFunction)) {
+          throw e;
```

Biome (2.1.2)

[error] Unsafe usage of 'throw'.

'throw' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)